### ISSUE TITLE
nuxt-i18n v6.13.5 incompatible with nuxt v2.13.3

### ISSUE THREAD

**Issue by coremyslo** [nuxt-modules/i18n#829]
### Version
nuxt-i18n: 6.13.5 (lower versions are fine)
nuxt: 2.13.3 (higher versions are fine)

#### [mode](https://nuxtjs.org/api/configuration-mode): 
 - [x] universal
 - [] spa

### Nuxt-i18n configuration

```
["nuxt-i18n", {
    differentDomains: true,
    defaultLocale: config.LOCALE_DEFAULT,
    locales,
    parsePages: false,
    pages,
    vueI18n: {
        fallbackLocale: config.LOCALE_FALLBACK,
        silentFallbackWarn: true,
    },
    vueI18nLoader: true,
    vuex: {
        syncMessages: true,
        syncLocale: true,
    },
    lazy: true,
    langDir: 'locales/'
}],
```

### What is actually happening?
<img width="566" alt="image" src="https://user-images.githubusercontent.com/20535974/89335166-55c05180-d6a0-11ea-99a6-361f7f8c24e2.png">

**Comment by rchl** [comment]
@coremyslo Can you post the `locales` object also?

Or even complete configuration as I'm not sure what part of the configuration is the culprit.

**Comment by coremyslo** [comment]
@rchl I also tried to remove things to see where is start to break. I've got something like this in the end.
```
["nuxt-i18n", {
    locales: [
        {
            code: 'fr-FR-local',
            iso: 'fr-FR',
            file: 'index.js',
            domain: 'my.domain.loc:5555'
        }
    ]
}],
```
UPDATE: actually even below fails with the same error
```
const nuxtConfig = {
    modules: [
        ["nuxt-i18n", {
            locales: [
                {
                    code: 'fr-FR-local',
                    iso: 'fr-FR',
                    file: 'index.js',
                    domain: 'my.domain.loc:5555'
                }
            ],
        }],
    ],
};

export default nuxtConfig;
```

**Comment by rchl** [comment]
Before I'm gonna do any further investigation, I'd like you to try delete `node_modules`, re-install dependencies and try again.

I'm starting to think you might be hitting a caching issue https://github.com/nuxt/nuxt.js/issues/7838 (which is gonna be fixed for next NuxtJS version).

**Comment by coremyslo** [comment]
Deleted `yarn.lock` and removed `node_modules`, called `yarn` and `yarn dev` -> same issue.

I'm just tried to create an empty project with
```
// package.json
{
  "name": "test",
  "scripts": {
    "dev": "nuxt"
  },
  "dependencies": {
    "nuxt": "2.13.3",
    "nuxt-i18n": "6.13.5"
  }
}
```

```
// nuxt.config.js
const nuxtConfig = {
    modules: [
        ["nuxt-i18n", {
            locales: [
                {
                    code: 'fr-FR-local',
                    iso: 'fr-FR',
                    file: 'index.js',
                    domain: 'my.domain.loc:5555'
                }
            ],
        }],
    ],
};

export default nuxtConfig;
```
It still reproduces for me.

node v14.4.0
yarn v1.22.4

**Comment by rchl** [comment]
OK, thanks. Actually yarn should be unaffected by that caching bug.

I've ignored the fact the you've mentioned that it happens in Nuxt v2.13.3 because I didn't use any new stuff from Nuxt v2.14.0 but yes, I can see it being broken in the older version. Will have a look.

**Comment by coremyslo** [comment]
thanks.

**Comment by rchl** [comment]
Fix released in [v6.13.6](https://github.com/nuxt-community/i18n-module/releases/tag/v6.13.6)
